src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509Store.cs

```diff
@@ -128,10 +128,14 @@ public void Add(X509Certificate2 certificate)
         {
             if (certificate == null)
                 throw new ArgumentNullException(nameof(certificate));
-
+            
```

Comment:
Please don't add spaces to an otherwise blank line.  (I'm sure the editor thought it was being helpful at the time)